### DISCUSSION TITLE
Screenshot is not getting added to allure report

### DISCUSSION THREAD

**Discussion by Shoaibsheikh786** [allure-framework/allure2#2443] [Bug Reports]
Hello
I can see screenshot in allure results folder but when i open the report with allure serve . It is not present in the allure report,

**Comment by ciconq** [comment]
I can confirm that locally I am seeing the screenshot being attached but with the same code when tests are running in Azure DevOps there is no screenshot attached in the Allure Report. Is it possible that there is an issue within the Allure Report Azure DevOps extension?

Did a quick demo example and this is how I am seeing my failed test body locally(as it should be I suppose):

![normal test body](https://github.com/allure-framework/allure2/assets/16039275/c09e44d5-b003-4795-a206-b8299582bdd7)

And this is how I see it currently in Azure DevOps:

![abnormal test body](https://github.com/allure-framework/allure2/assets/16039275/94c2ee04-2e40-40aa-ae5f-4f3e16126b93)

**Comment by ciconq** [comment]
Are there any updates on this? I can see a new version released for the Azure DevOps Allure Report extension today but the Test body is still broken and no Screenshots and Console Output sections are available within it(Test body) as per the screenshots above.

**Reply by epszaw** [reply]
Hey there! There wasn't any fix for the problem. Which test framework do you use?

**Reply by ciconq** [reply]
Hey! I am using NUnit(latest 4.1.0) along with latest NUnit3TestAdapter(4.5.0). Just tried to see if the problem is related to the VSTest@3 task where I am rerunning failed tests but with this feature being disabled I am not able to see the screenshots as well and previously I can confirm that it was working with this same chunk of code:

```

    [TearDown]
    public void QuitDriver()
    {
        if (TestContext.CurrentContext.Result.Outcome != ResultState.Success)
        {
            var content = ((ITakesScreenshot)_driver).GetScreenshot().AsByteArray;
            AllureApi.AddAttachment("Failed screenshot", "image/png", content);
        }

        _driver?.Quit();
    }
    
```

**Reply by epszaw** [reply]
Could you please try to generate the report locally with `--single-file` flag and check screenshots once more?

**Reply by ciconq** [reply]
Locally it is working as expected when running `allure generate "allure-results" --clean --single-file`:

![single report](https://github.com/allure-framework/allure2/assets/16039275/7899a4a2-2e8a-4069-9082-0cda09e34f3d)

It is also working without the `--single-file` option(locally).

**Reply by ciconq** [reply]
Hey, are there any updates on this issue?

**Reply by ciconq** [reply]
Hey @epszaw, are there any updates on this issue?

**Comment by LuvDeluxe** [comment]
I can reproduce this bug in Azure DevOps as well(locally it works fine). Are the any updates on the issue?
